base.cfg

```diff
@@ -15,10 +15,10 @@ parts +=
     check-readme
 develop = .
 
-# auto-checkout = kitconcept.recipe.solr
+auto-checkout = plone.restapi
 
 [sources]
-plone.restapi = git https://github.com/plone/plone.restapi.git
+plone.restapi = git https://github.com/plone/plone.restapi.git branch=python3_updated
```

Comment:
@reinhardt could you elaborate why we use the python3_updated branch here? That branch has been merged into plone.restapi long time ago and all recent releases should be python3 compatible.

Reply:
I originally thought plone.restapi master was not python 3 compatible, but at the second glance it looks more like collective.solr imports need some adaptation to work with the latest plone.restapi code.

Reply:
@reinhardt yeah. We refactored some internals in plone.restapi. Just pin plone.restapi to 3.9.0: https://github.com/collective/collective.solr/commit/50f49e50997620f72f8ea6daea78a8a57392499b

We can look into the plone.restapi upgrade later...